src/pages/pricing.js

```diff
@@ -154,6 +154,109 @@ const PricingPage = () => {
                                 </Card>
                             </Col>
                         ))}
+                        {state.planOptions === 'vpc' && (
+                            <div className="pricing-cloud">
+                                <h4>Much less expensive for higher volume usage. Pay just for what you use.</h4>
+                                <div>
+                                    Our pricing is logarithmic and gets much, much less expensive at scale with VPC
+                                    deployment. No setup fee or minimum commitment.
```

Comment:
Hm I'm not sure no minimum commitment will ever make sense for us. I think we'll automate a large amount but there's still some overhead/work for each individual acount, and i think we'll be annoyed if we're spending a lot of time on a $100/month account.

Reply:
> Should we include potential infrastructure costs here?
> 

Yep, out of scope for this PR (just to make it correct), but will quickly address that as I'm getting asked it repeatedly.